src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -213,10 +213,9 @@ public override bool HasConditionalStaticDependencies
 
                 // Since the vtable is dependency driven, generate conditional static dependencies for
                 // all possible vtable entries
-                foreach (MethodDesc method in _type.GetMethods())
+                if (_type.GetClosestMetadataType().GetAllVirtualMethods().GetEnumerator().MoveNext())
```

Comment:
We don't appear to use much LINQ in the compiler. Can I do `_type.GetClosestMetadataType().GetAllVirtualMethods().Any()`?